webapp/portlet/src/main/webapp/common/components/ExoIdentitySuggester.vue

```diff
@@ -182,6 +183,12 @@ export default {
         return null;
       },
     },
+    innerIcon: {
```

Comment:
We will have to differentiate the display of the suggester input by using an explicit and meaningful variable name like `filterStyle` prop of type boolean that will have `false` as default value.